packages/pigeon/platform_tests/test_plugin/example/ios/RunnerTests/AllDatatypesTests.swift

```diff
@@ -40,10 +41,10 @@ class AllDatatypesTests: XCTestCase {
       anInt: 1,
       aDouble: 2.0,
       aString: "123",
-      aByteArray: [UInt8]("1234".data(using: .utf8)!),
-      a4ByteArray: [Int32].init(arrayLiteral: 1, 2, 3, 4),
-      a8ByteArray: [Int64].init(arrayLiteral: 1, 2, 3, 4, 5, 6, 7, 8),
-      aFloatArray: [Float64].init(arrayLiteral: 1, 2, 3, 4, 5, 6, 7, 8),
+      aByteArray: [UInt8]("1234".data(using: .utf8)!) as? FlutterStandardTypedData,
+      a4ByteArray: [Int32].init(arrayLiteral: 1, 2, 3, 4) as? FlutterStandardTypedData,
+      a8ByteArray: [Int64].init(arrayLiteral: 1, 2, 3, 4, 5, 6, 7, 8) as? FlutterStandardTypedData,
+      aFloatArray: [Float64].init(arrayLiteral: 1, 2, 3, 4, 5, 6, 7, 8) as? FlutterStandardTypedData,
```

Comment:
I'm pretty sure these are just `nil` since you are `as?`-casting to a different type rather than constructing `FlutterStandardTypedData`s.